cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1667,9 +1667,10 @@
     <name>security.auth.server.announce.urls</name>
     <value></value>
     <description>
-      CDAP Authentication service announce URL's separated by comma. Each URL is in the format of protocol://host:port.
-      These are the URL's that clients should use to communicate with the Authentication Server.
-      Leave empty to use the default value generated by the Authentication Server.
+      A comma-separated list of CDAP Authentication service announce URLs. Each URL is in
+      the form of protocol://host:port. These are the URLs that clients should use to
+      communicate with the Authentication Server. Leave empty to use the default value
+      generated by the Authentication Server.
```

Comment:
Maybe also mention `If deprecated property "security.auth.server.announce.address" is set and this property is empty, "security.auth.server.announce.address" will be used`